src/canvascontext.js

```diff
@@ -13,6 +15,26 @@ export class CanvasContext {
   static get HEIGHT() {
     return 400;
   }
+  static get CANVAS_BROWSER_SIZE_LIMIT() {
+    return 32767; // Chrome/Firefox. Could be determined more precisely by npm module canvas-size
+  }
+
+  static CheckCanvasDimensionsForBrowserLimit(width, height) {
+    if (Math.max(width, height) > this.CANVAS_BROWSER_SIZE_LIMIT) {
+      Vex.W(
+        'canvas dimensions exceed browser limit. reducing/cropping to ' +
```

Comment:
Fix capitalization please 'Canvas dimensions exceed browser limit. Cropping to ...'

Reply:
done